tests/docker/config/storage.conf

```diff
@@ -0,0 +1,34 @@
+app {
```

Comment:
Changed the way to run the storage in the tests so that it aligns with Delta
More readable than using system properties too